golang/wasm/pure.js

```diff
@@ -184,3 +184,30 @@ function getKeysForDomain() {
 
 	return
 }
+
+// findDOHEndpoint()
+// for a given domain, find DOH Endpoint URL for update returned as string
+//
+async function findDOHEndpoint() {
+	var dohDomain = document.getElementById("doh-for-domain").value
+	if (! dohDomain.endsWith('.')) {
+		dohDomain = dohDomain + '.'
+	}
+
+	const div = document.getElementById("domain-doh-endpoint")
+	if (div.children.length > 0) {
```

Comment:
not sure if it is possible to have more then one but instead of removing the first you can also just `div.innerHTML = ""`, i think.